LaTeX/LaTeX.sublime-syntax

```diff
@@ -552,7 +552,7 @@ contexts:
         4: variable.parameter.function.latex
         5: punctuation.definition.group.brace.end.latex
       push:
-        - meta_scope: meta.environment.math.block.be.latex
+        - meta_content_scope: meta.environment.math.block.be.latex
```

Comment:
For consistency, shall this also be applied to `\[\]` and `\(\)` and `$$` and `$` pairs?
Though, it may break BracketHighlighter or make BracketHighlighter hard to highlight brackets.